### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by sideeffffect**

### CODE REVIEW

**Comment by sideeffffect** [comment]
@cquiroz :pray: :point_up:

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,105 +15,87 @@ on:
     tags: [v*]
 
 env:
-  PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
```
Comment (cquiroz) [inline .github/workflows/ci.yml:18]: These need to be included to do publication I think

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -125,123 +107,150 @@ jobs:
...
         run: |
-          echo "$PGP_SECRET" | base64 -di > /tmp/signing-key.gpg
+          echo "$PGP_SECRET" | base64 -d -i - > /tmp/signing-key.gpg
           echo "$PGP_PASSPHRASE" | gpg --pinentry-mode loopback --passphrase-fd 0 --import /tmp/signing-key.gpg
           (echo "$PGP_PASSPHRASE"; echo; echo) | gpg --command-fd 0 --pinentry-mode loopback --change-passphrase $(gpg --list-secret-keys --with-colons 2> /dev/null | grep '^sec:' | cut --delimiter ':' --fields 5 | tail -n 1)
 
       - name: Publish
-        run: sbt '++ ${{ matrix.scala }}' tlRelease
+        env:
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```
Comment (cquiroz) [inline .github/workflows/ci.yml:220]: Ah I see they are defined here

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,11 +1,11 @@
-addSbtPlugin("org.portable-scala" % "sbt-crossproject" % "1.2.0")
+addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.3.2")
 
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.12.0")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.16.0")
 
 addSbtPlugin("io.github.cquiroz" % "sbt-tzdb" % "4.2.0")
 
-addSbtPlugin("org.typelevel" % "sbt-typelevel" % "0.4.17")
+addSbtPlugin("org.typelevel" % "sbt-typelevel-ci-release" % "0.7.1")
```
Comment (cquiroz) [inline project/plugins.sbt:7]: great, thanks

**Comment by cquiroz** [comment]
Thanks this is great,  do you kow why one of the build steps doesn't complete?

File: sbt [hunk 4]
```diff
@@ -0,0 +1,664 @@
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (sideeffffect) [reply sbt:1]: The macOS image doesn't seem to contain sbt. So we include this bash wrapper script which make sbt startable from anywhere (anywhere bash is).

**Pull request merged**